README.md

```diff
@@ -23,11 +23,14 @@ Fabric React is a responsive, mobile-first collection of robust components desig
 
 ## View the docs
 
-Before you get started, make sure you have [node.js](https://nodejs.org/), [gulp](http://gulpjs.com/), and [git](https://git-scm.com/) installed. To view the documentation including examples, contracts, component status, and to add functionality or fix issues locally, you can:
+Before you get started, **make sure you have [node.js](https://nodejs.org/) and [git](https://git-scm.com/) installed.**
 
-1. `git clone https://github.com/OfficeDev/office-ui-fabric-react.git`
-2. `npm install`
-3. `npm start`
+To view the documentation including examples, contracts, component status, and to add functionality or fix issues locally, you can:
+
+1. `npm install -g gulp`

```

Comment:
is global gulp actually required? are we not just using a locally installed copy?